main.go

```diff
@@ -106,6 +107,18 @@ func discordMainWrapper() error {
 		url = DefaultURL
 	}
 
+	extPort := os.Getenv("EXT_PORT")
+	if extPort == "" {
+		log.Print("[Info] No EXT_PORT provided. Defaulting to PORT\n")
+	} else if extPort == "protocol" {
+		log.Print("[Info] EXT_PORT set to protocol. Not adding port to url\n")
+	} else {
+		num, err := strconv.Atoi(extPort)
+		if err != nil || num < 1024 || num > 65535 || num == 80 || num == 443 {
```

Comment:
I don't think the logic is right here; we want to reject all ports < 1024 or > 65535, EXCEPT for 80/443, right? So I think the logic should be `num > 65535 || (num < 1024 && num != 80 && num != 443)`